pkg/apis/sources/v1alpha1/vspheresource_types.go

```diff
@@ -41,7 +41,13 @@ var _ kmeta.OwnerRefable = (*VSphereSource)(nil)
 type VSphereSourceSpec struct {
 	duckv1.SourceSpec `json:",inline"`
 
-	VAuthSpec `json:",inline"`
+	VAuthSpec        `json:",inline"`
+	CheckpointConfig VCheckpointSpec `json:"checkpointConfig"`
```

Comment:
Since this is in the Spec now, I think it might be better to add defaults here:
https://github.com/vmware-tanzu/sources-for-knative/blob/master/pkg/apis/sources/v1alpha1/vspherebinding_defaults.go#L13

So, the user can see it by inspecting the resource. Also, probably add some validation here for the new values:
https://github.com/vmware-tanzu/sources-for-knative/blob/master/pkg/apis/sources/v1alpha1/vspherebinding_validation.go#L30

Reply:
Done.